client/public/index.html

```diff
@@ -4,31 +4,27 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    <!-- third-generation iPad with high-resolution Retina display: -->
     <link
-      rel="apple-touch-icon-precomposed"
-      sizes="144x144"
-      href="%PUBLIC_URL%/favicon144.png"
+      rel="shortcut icon"
+      href="%PUBLIC_URL%/favicon.ico"
+      type="image/x-icon"
     />
```

Comment:
I don’t think we even need to specify this. If the browser does not support all the other formats we specify below, it will eventually go look at the root of the website for a `favicon.ico`

Reply:
In https://github.com/mdn/yari/pull/3621#pullrequestreview-644934488 it was recommended to drop the `shortcut` value for `rel`, too.

Reply:
Lemme merge in `main` once https://github.com/mdn/yari/pull/3621 lands. 
This PR is supposed to be about removing the `favicon144.png` image.